tests/FakeItEasy.Specs.netstd/project.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "FakeItEasy.Specs",
+  "version": "1.0.0-*",
+  "description": "FakeItEasy.Specs.netstd Class Library",
+  "authors": [ "" ],
+  "packOptions": {
+    "tags": [ "" ],
+    "projectUrl": "",
+    "licenseUrl": ""
+  },
+
+  "buildOptions": {
+    "keyFile": "../../FakeItEasy.snk",
+    "compile": {
+      "include": [
+        "../FakeItEasy.Specs/**/*.cs"
+      ],
+      "exclude": [
+        "../FakeItEasy.Specs/SelfInitializedFakesSpecs.cs",
+        "../FakeItEasy.Specs/DisposableSpecs.cs"
+      ]
+    },
+    "define": [ "NETCORE" ]
+  },
+
+  "dependencies": {
+    "xunit.abstractions": "2.0.1-rc2",
+    "xunit.extensibility.core": "2.2.0-beta2-build3300",
+    "xunit.extensibility.execution": "2.2.0-beta2-build3300"
+  },
+
+  "testRunner": "xunit",
+  "frameworks": {
+    "netcoreapp1.0": {
+      "buildOptions": {
+        "define": [ "NETCORE", "FEATURE_NETCORE_REFLECTION" ]
+      },
+      "dependencies": {
+        "Castle.Core": "4.0.0-beta002",
+        "FakeItEasy": {
+          "target": "project",
+          "version": "99.99.99-wrapped"
+        },
+        "FakeItEasy.Tests.netstd": {
+          "target": "project",
+          "version": "1.0.0"
+        },
+        "FluentAssertions": "4.6.3",
+        "Microsoft.NETCore.App": {
+          "type": "platform",
+          "version": "1.0.0"
+        },
+        "System.Console": "4.0.0",
+        "System.Diagnostics.TraceSource": "4.0.0",
+        "System.Linq.Expressions": "4.1.0",
+        "System.Reflection": "4.1.0",
+        "System.Reflection.Emit.ILGeneration": "4.0.1",
+        "System.Reflection.Extensions": "4.0.1",
+        "System.Reflection.TypeExtensions": "4.1.0",
+        "System.Runtime.Serialization.Primitives": "4.1.1",
+        "System.Threading.Tasks.Parallel": "4.0.1",
+        "Microsoft.CSharp": "4.0.1",
+        "dotnet-test-xunit": "2.2.0-preview2-build1029",
+        "Xbehave.Core": "2.2.0-beta0001-build680"

```

Comment:
Some of these dependencies are probably not necessary; is there an easy way to detect unused dependencies?